akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -542,3 +545,21 @@ Scala
 
 Java
 :  @@snip [StreamAttributeDocTest.java](/akka-stream-tests/src/test/java/akka/stream/StreamAttributeDocTest.java) { #attributes-on-stream }
+
+### Stream cancellation available upstream
+
+Previously an Akka streams stage or operator failed it was impossible to discern this from 
+the stage just cancelling. This has been improved so that when a stream stage fails the cause
+will be propagate upstream.
+
+The following operators has a slight change in behavior because of this:
+
+* `FileIO.fromPath`, `FileIO.fromFile` and `StreamConverters.fromInputStream`  will fail the materialized future with 
+  an `IOOperationIncompleteException` when downstream fails
+* `.watchTermination` will fail the materialized `Future` or `CompletionStage` rather than completing it when downstream fails
+* 
+
+This also means that custom `GraphStage` implementations should be changed to pass the
```

Comment:
Maybe change to "pass on the cancellation cause"? (Seems both is valid in English with a preference on keeping verb and preposition closer together. In German, it is required to put the preposition at the end of the sentence so I'm used to that but in English it seems a bit confusing ;) )